### PR TITLE
Adding example with-fingerprintjs-pro for identifying visitors

### PR DESCRIPTION
**Pull request by ilfa**
## Documentation / Examples

- [x] Make sure the linting passes by running `pnpm lint`
- [x] The examples guidelines are followed from [our contributing doc](https://github.com/vercel/next.js/blob/canary/contributing.md#adding-examples)

Deployed on https://with-fingerprintjs-pro.vercel.app/

### CODE REVIEW

File: examples/with-fingerprintjs-pro/pages/index.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import type { NextPage } from 'next'
```
Comment (balazsorban44) [inline examples/with-fingerprintjs-pro/pages/index.tsx:1]: I wonder why this page is needed?

File: examples/with-fingerprintjs-pro/package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "next": "latest",
+    "react": "18.1.0",
+    "react-dom": "18.1.0"
+  },
+  "devDependencies": {
+    "@types/node": "17.0.40",
+    "@types/react": "18.0.12",
+    "eslint": "8.17.0",
+    "eslint-config-next": "latest",
+    "typescript": "4.7.3"
```
Comment (balazsorban44) [inline examples/with-fingerprintjs-pro/package.json:20]:
```suggestion
    "start": "next start"
  },
  "dependencies": {
    "@fingerprintjs/fingerprintjs-pro-react": "1.0.1",
    "next": "latest",
    "react": "18.1.0",
    "react-dom": "18.1.0"
  },
  "devDependencies": {
    "@types/node": "17.0.40",
    "@types/react": "18.0.12",
    "typescript": "4.7.3"
```

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Thanks, made a small suggestion to comply with our [contribution guidelines](https://github.com/vercel/next.js/blob/canary/contributing.md#adding-examples)

File: examples/with-fingerprintjs-pro/pages/index.tsx (hunk 1 above)
Comment (ilfa) [reply examples/with-fingerprintjs-pro/pages/index.tsx]: Thanks, fixed! Didn't know about `redirects` section for `next.config.js` before.

**Comment by ilfa** [comment]
Hello @balazsorban44!
Do I need to update the branch?

**Review by balazsorban44 — APPROVED** [review]
Looks good to me now, thank you!

**Pull request merged**